src/ASM/ASMs3D.C

```diff
@@ -1716,7 +1716,7 @@ void ASMs3D::getBoundaryNodes (int lIndex, IntVec& nodes,
 {
   if (basis < 1) basis = 1;
 
-  if (!this->getBasis(basis)) return; // silently ignore empty patches
+//  if (!this->getBasis(basis)) return; // silently ignore empty patches
```

Comment:
Then remove the line entirely instead. Similar for ASMs2D I guess.

Reply:
done.